DESCRIPTION

```diff
@@ -11,12 +11,12 @@ Encoding: UTF-8
 LazyData: true
 Roxygen: list(markdown=TRUE)
 RoxygenNote: 7.3.1
+Depends:
+    R (>= 4.0)
```

Comment:
just curious, where does this version dep come from?

Reply:
I added it since it seems to be good practice to have a minimum R version to depend on. I thought at minimum 4.0 would be good since R is at 4.4 and RStudio requires at least 3.6.